docs/couchdb.md

```diff
@@ -0,0 +1,29 @@
+# Procedure to create a remote couchdb database
+
+In order to run element-api in production you will need to define a `ELEMENT_COUCHDB_REMOTE` in your `.env` file. Here is how you can obtain one for free (up to 1GB of data)
+
+## Registering an IBM Cloud account
```

Comment:
We should add there are other ways to host CouchDB including using docker / running your own, but this cloud instance is free.